### PR TITLE
Update Release Notes 5.3.2

### PR DESCRIPTION
**Pull request by tst-nfarah**
This address #1462

### CODE REVIEW

File: docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_2.rst [hunk 1]
```diff
@@ -80,9 +85,27 @@ Release notes: Open GEE 5.3.2
...
-           - The current patch releases have been incorporated.
+         * - 1376
+           - Portable Globe Cutter fails the last step when using HTTPS
+           - Used the advanced configuration to set a URL that will be used when cutter tries to make an openurl call.            
+         * - 1398
+           - System manager crashed because of casting asset version to asset
+           - Fixed the invalid cast.
+         * - 1401
+           - No notification when cache size is exceeded causing performance issues
+           - Added Loging record when cache size is exceeded.
```
Comment (tst-swanson) [inline docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_2.rst]: Should this be logging?

File: docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_2.rst [hunk 2]
```diff
@@ -12,6 +12,11 @@ Release notes: Open GEE 5.3.2
 
       .. rubric:: New Features
 
+      **Documentation overhaul**. All documentation were migrated to be
```
Comment (tst-swanson) [inline docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_2.rst]: All documentation was migrated to Sphinx format, making it easier to edit and maintain. User documentation file names were also migrated from a number-based naming convention to self-explanatory descriptive titles.

**Comment by tst-nfarah** [comment]
> it would be nice if you could get the latest 5_3.2 changes in and do make html again. As we have made changes in Makefile so that all warnings are now treated as errors. We can catch any syntax issues in rst while doing make.

This would be nice to have but not part of this PR ? Is it ?

**Comment by tst-hrajamoney** [comment]
> > it would be nice if you could get the latest 5_3.2 changes in and do make html again. As we have made changes in Makefile so that all warnings are now treated as errors. We can catch any syntax issues in rst while doing make.
> 
> This would be nice to have but not part of this PR ? Is it ?
It should be part of this PR, as it will help verify the rst syntax of the changes made. 
I notice that you have committed only the relevant files. It is better to add files that git shows under 5.3.3 . As I have seen, Sphinx generates: docs/geedocs/5.3.3/objects.inv, , docs/geedocs/5.3.3/searchindex.js. 
what i do in cmd line is git add 5.3.3/* , which adds all the modified files

**Comment by tst-nfarah** [comment]
ok. I now committed all touched files by the Sphinx build.

File: docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_2.rst (hunk 2 above)
Comment (tst-nfarah) [reply docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_2.rst]: change was now made.

File: docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_2.rst (hunk 1 above)
Comment (tst-nfarah) [reply docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_2.rst]: > Should this be logging?

Yes. pushed the edit.

**Comment by tst-nfarah** [comment]
Closing this PR to replace it with this [other PR](https://github.com/google/earthenterprise/pull/1467). Documentation changed creating conflicts that was not obvious to fix.

**Pull request closed without merging**